### PR TITLE
gcta --> added start_dir due to live build failing

### PR DESCRIPTION
**Pull request by afraz-ahmed**
For INC1042557

Update for #217 

gcta-1.93.2-foss-2019b.eb
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] Ubuntu16 VM

### CODE REVIEW

**Comment by edmondac** [comment]
@branfosj could you review this, since I've changed it?

File: easybuild/easyconfigs/g/gcta/gcta-1.93.2-foss-2019b.eb [hunk 1]
```diff
@@ -14,10 +14,18 @@ description = """GCTA (Genome-wide Complex Trait Analysis) was originally design
...
-checksums = ['fc2845c0b782443c6b5999d1c8cfe332adb9ddbd379dfff11d3b4e80cd8e9f29']
-
+source_urls = [
+    'https://cnsgenomics.com/software/gcta/pre_gcta',
+    'https://github.com/zhilizheng/plink-ng/archive',
+]
+sources = [
+    '%(name)s_%(version)s_src.zip',  # gcta
+    '%s.zip' % _plink_ng_commit,  # plink-ng
+]
```
Comment (branfosj) [inline easybuild/easyconfigs/g/gcta/gcta-1.93.2-foss-2019b.eb:24]:
```suggestion
source_urls = ['https://cnsgenomics.com/software/gcta/pre_gcta']
sources = [
    '%(name)s_%(version)s_src.zip',  # gcta
    {
        'source_urls': ['https://github.com/zhilizheng/plink-ng/archive'],
        'download_filename': 'ec1b2d151cb77050b5ba09dc28157a03fc3cc968.tar.gz',
        'filename': 'plink-ng-20191208.tar.gz',
        'extract_cmd': 'tar -C %(name)s-master/submods/plink-ng --strip-components=1 -xf %s',
    }
]
```

File: easybuild/easyconfigs/g/gcta/gcta-1.93.2-foss-2019b.eb [hunk 2]
```diff
@@ -37,15 +45,12 @@ dependencies = [
 preconfigopts = 'export EIGEN3_INCLUDE_DIR=$EBROOTEIGEN/include && '
 preconfigopts += 'export SPECTRA_LIB=$EBROOTSPECTRA/include && '
 preconfigopts += 'export BOOST_LIB=$EBROOTBOOST && '
-preconfigopts += 'git clone https://github.com/zhilizheng/plink-ng submods/plink-ng && '
-preconfigopts += 'cd submods/plink-ng && git checkout %s && cd ../.. && ' % _plink_ng_commit
-
-configure_cmd = 'mkdir build && cd build && cmake ..'
-build_cmd = 'cd build && make'
+preconfigopts += ('(cd %%(builddir)s/%%(name)s-master/submods/ && rmdir plink-ng && '
+                  ' mv ../../plink-ng-%s plink-ng) && ' % _plink_ng_commit)
```
Comment (branfosj) [inline easybuild/easyconfigs/g/gcta/gcta-1.93.2-foss-2019b.eb:49]:
```suggestion
```

File: easybuild/easyconfigs/g/gcta/gcta-1.93.2-foss-2019b.eb [hunk 3]
```diff
@@ -14,10 +14,18 @@ description = """GCTA (Genome-wide Complex Trait Analysis) was originally design
...
+    'https://cnsgenomics.com/software/gcta/pre_gcta',
+    'https://github.com/zhilizheng/plink-ng/archive',
+]
+sources = [
+    '%(name)s_%(version)s_src.zip',  # gcta
+    '%s.zip' % _plink_ng_commit,  # plink-ng
+]
+checksums = [
+    'fc2845c0b782443c6b5999d1c8cfe332adb9ddbd379dfff11d3b4e80cd8e9f29',  # gcta_1.93.2_src.zip
+    'e55270b66ae3ce82319457e372f83e03bd11b1e29cce5b550b3cefa31d142eba',  # ec1b2d151cb77050b5ba09dc28157a03fc3cc968.zip
```
Comment (branfosj) [inline easybuild/easyconfigs/g/gcta/gcta-1.93.2-foss-2019b.eb:27]:
```suggestion
    '5734cd20f69195d897fcf5702355d32f53b0095c6a798f3dda1c40768c34b53d',  # plink-ng-20191208.tar.gz
```

**Comment by afraz-ahmed** [comment]
Can this line be removed from build dependencies? `('git', '2.23.0', '-nodocs'),`

**Pull request merged**